coderd/database/migrations/000135_drop_rich_parameter_legacy_variable.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE template_version_parameters DROP COLUMN legacy_variable_name;
```

Comment:
What could happen if legacy variables are still present when we upgrade? Should we somehow prevent this migration if any column has a non-empty/null value here?

I'm not sure what we should do, but perhaps something like this (granted the condition is needed).

```sql
do $$
  BEGIN
    RAISE 'Bad things will happen!';
  END;
$$ LANGUAGE plpgsql;
```

Reply:
Legacy variable names are not used since 2 minor releases (v0.24.0, v0.25.0). I would assume that we can just drop the column :) Users who forgot to migrate workspaces will not benefit from the `legacy_variable_name` property.